<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_DimensionLabelEditor</class>
 <widget class="QWidget" name="QG_DimensionLabelEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>124</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>220</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dimension Label Editor</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="bgLabel">
     <property name="title">
      <string>Dimension Label:</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="lLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text">
             <string>Label:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="bDiameter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../res/extui/extui.qrc">
              <normaloff>:/extui/char_diameter.png</normaloff>:/extui/char_diameter.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="leTol1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leTol2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel1">
          <property name="text">
           <string>Insert:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbSymbol">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>⌀ (Diameter)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>° (Degree)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>± (Plus / Minus)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>π (Pi)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>× (Times)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>÷ (Division)</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../res/extui/extui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbSymbol</sender>
   <signal>activated(QString)</signal>
   <receiver>QG_DimensionLabelEditor</receiver>
   <slot>insertSign(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
